xtask/Cargo.toml

```diff
@@ -10,8 +10,12 @@ clap = { version = "4.0.4", features = ["derive"] }
 # The latest fatfs release (0.3.5) is old, use git instead to pick up some fixes.
 fatfs = { git = "https://github.com/rafalh/rust-fatfs.git", rev = "87fc1ed5074a32b4e0344fcdde77359ef9e75432" }
 fs-err = "2.6.0"
+heck = "0.4.0"
 mbrman = "0.5.0"
 nix = "0.25.0"
+proc-macro2 = "1.0.46"
+quote = "1.0.21"
 regex = "1.5.4"
 serde_json = "1.0.73"
+syn = { version = "1.0.101", features = ["full"] }
```

Comment:
Considering many people won't be usually interested in (re-)generating or checking the generated code, would it be a good idea to move these new dependencies behind an optional feature, and only enable it when we actually need it? AFAIK, `syn` and `quote` can take a bit to compile.

Reply:
These deps are only in `xtask`, so they won't impact end users. And we already depend on syn and quote in uefi-macros, so no change anyway.

(I do think making those deps optional for end users is a good goal, just outside the scope of this PR.)

Reply:
Ah, correct, I forgot that you don't need to compile `xtask` when you're a downstream user of the crate 😅